src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PEMethodSymbol.vb

```diff
@@ -641,7 +641,26 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
             If Not _packedFlags.IsCustomAttributesPopulated Then
                 Dim attributeData As ImmutableArray(Of VisualBasicAttributeData) = Nothing
                 Dim containingPEModuleSymbol = DirectCast(ContainingModule(), PEModuleSymbol)
-                containingPEModuleSymbol.LoadCustomAttributes(Me.Handle, attributeData)
+                Dim checkForRequiredMembers = MethodKind = MethodKind.Constructor AndAlso
```

Comment:
See https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs#L1012-L1030 for the equivalent C# version of this. I based this logic on the current C# code.